independent-publisher-connectors/WebEx/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+          "identityProvider": "oauth2",
+          "clientId": "C5dd1ba47c91e7a80dd71a82f465918bf0e7b9ff260162f623ef583a06123c641",
```

Comment:
@ocarhui secret look alike values are not allowed, please replace the clientId with a dummy value. For example "Please enter your client ID".